default.nix

```diff
@@ -317,17 +317,23 @@ in rec {
             in allConfig;
         in (mkProject (projectDefinition args)).projectConfig);
       mainProjectOut = projectOut { inherit system; };
-      serverOn = projectInst: bundledConfig: version: serverExe
-        projectInst.ghc.backend
-        mainProjectOut.ghcjs.frontend
-        projectInst.passthru.staticFiles
-        projectInst.passthru.__closureCompilerOptimizationLevel
-        bundledConfig
-        version;
-      linuxExe = serverOn (projectOut { system = "x86_64-linux"; }) null;
-      ovaExe = serverOn (projectOut { system = "x86_64-linux"; }) (base' + /config);
+      serverOn = { projectInst, bundledConfig ? null, version } : serverExe {
+        inherit bundledConfig version;
+        inherit (projectInst.ghc) backend;
+        inherit (mainProjectOut.ghcjs) frontend;
```

Comment:
`mainProjectOut != projectInst` in `linuxExe` (and consequently in `ovaExe`). Is the extra indirection undesirable only for this attribute?